src/app/modules/submission/multirespondent-submission/multirespondent-submission.middleware.ts

```diff
@@ -195,9 +192,8 @@ const asyncVirusScanning = (
   enableGuarddutyLambdaInvoke: boolean | undefined,
 ): ResultAsync<
   IdTaggedParsedClearAttachmentResponseV3,
-  | VirusScanFailedError
```

Comment:
Use TS type constants instead of hardcoding so that the upstream functions are updated when the error types change - to avoid missed error types